### DISCUSSION TITLE
[Feature] Button to download all the pictures of a person

### DISCUSSION THREAD

**Discussion by ManuXD32** [immich-app/immich#11360] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add a button to download all the photos associated with a person.

Sometimes if you have a lot of photos of someone, they cannot all be loaded al the same time, hence, it's difficult to download all of them at once.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by reneil1337** [comment]
yes please! this would be an incredible feature

**Comment by photoserver-pro** [comment]
Yes, it‘s an important Feature also for me

**Comment by johnmclane99** [comment]
This would be nice to have; also would be nice if it could have an option to download and keep the folder structure attached of where the photos were originally stored.
